build.sbt

```diff
@@ -2,19 +2,18 @@ import scala.collection.immutable.Seq
 import ReleaseTransformations._
 
 lazy val Versions = new {
-  val akka              = "2.4.12" // First version cross-compiled to 2.12
-  val akkaDns           = "2.4.2"
-  val dispatch          = "0.13.2"
-  val lagom13           = "1.3.0"
-  val lagom14           = "1.4.0-M2"
-  val play25            = "2.5.0"
-  val play26            = "2.6.0"
-  val scala211          = "2.11.11"
-  val scala212          = "2.12.3"
-  val scalaJava8Compat  = "0.8.0"
-  val scalaTest         = "3.0.1"
-  val sprayJson         = "1.3.3"
-  val typesafeConfig    = "1.3.1"
+  val akka                      = "2.4.12" // First version cross-compiled to 2.12
+  val akkaDns                   = "2.4.2"
```

Comment:
I noticed that akka dns isn't released to maven central btw, is this a problem for users of this lib or who uses it?

Reply:
Ideally I'd like to reorganize the akka-management project a bit as I proposed in email (having a core subproject with minimal dependencies, marked as provided) and provide this as a shaded dependency inside of reactive-lib.